config/locales/server.en.yml

```diff
@@ -1128,7 +1128,7 @@ en:
     body_min_entropy: "The minimum entropy (unique characters, non-english count for more) required for a post body."
     allow_uppercase_posts: "Allow all caps in a topic title or a post body."
 
-    title_fancy_entities: "Convert common ASCII characters to fancy HTML entities in topic titles, ala SmartyPants http://daringfireball.net/projects/smartypants/"
```

Comment:
Ah, "ala" as a colloquial form of "[a la](https://en.wiktionary.org/wiki/a_la#Alternative_forms)". Hmm.